sdk/tests/conformance/textures/misc/texture-formats-test.html

```diff
@@ -83,9 +83,7 @@
       var err = gl.getError();
       if (err == gl.NO_ERROR) {
           testFailed("should NOT be able to create texture of type " + formatName);
-      } else if (err == gl.INVALID_OPERATION) {
-          testFailed("should return gl.INVALID_ENUM for type " + formatName);
-      } else if (err == gl.INVALID_ENUM) {
+      } else if (err == gl.INVALID_ENUM || err == gl.INVALID_OPERATION) {
```

Comment:
These should only generate INVALID_ENUM or INVALID_VALUE because neither format or internalformat is legal, though they're both the same, per https://www.khronos.org/opengles/sdk/docs/man/xhtml/glTexImage2D.xml .